backend/internal/project/app_config_test.go

```diff
@@ -0,0 +1,92 @@
+package project
+
+import (
+	"net/url"
+	"path/filepath"
+	"testing"
+)
+
+func TestAppConfigLoad(t *testing.T) {
+	projectPath := t.TempDir()
+	err := createProjectStructure(projectPath, map[string]string{
+		// TODO: Weird things happen when passing in `file:app2/robin.app.json`
```

Comment:
Did you try `file://...`? `file:...` is not a valid URL

Reply:
I tried both, the thing is that in both cases, the URL parse succeeds but the opening fails.